tests/functional/recovery_key.js

```diff
@@ -50,6 +56,11 @@ registerSuite('Recovery key', {
 
       .then(openPage(SETTINGS_URL, selectors.SETTINGS.HEADER))
       .then(testElementExists(selectors.SETTINGS.HEADER))
+
+      // Perform a slight sleep here to ensure that the checks
+      // to enable the panel have been performed.
+      .sleep(500)
```

Comment:
Noticed that this these tests were a little flaky in circle ci. Testing it locally against circle, I able to intermittently reproduce the error. A visual inspection while test was running, there seems to be a slight delay before the setting panel appears. With this delay I couldn't reproduce error.